spec/models/category_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe Category do
+
+	let(:category) { build(:category) }
+	let(:subcategory) { build(:subcategory_1) }
+
+  it "has a valid factory" do
+    expect(category).to be_valid
+    expect(subcategory).to be_valid
+  end
+
```

Comment:
dopisz sprawdzenie czy relacje są poprawnie skonstruwone, przykład poniżej, i zastosować do każdego modelu gdzie są relacje
`

context 'model assiociations' do
--
  | it { should belong_to(:purchase_contractor) }
  | it { should belong_to(:sale_contractor) }
  | it { should belong_to(:auction) }
  | it { should have_many(:cart_fees) }
  | it { should accept_nested_attributes_for(:cart_fees) }
  | end

`